pooltasks.md

```diff
@@ -41,6 +43,13 @@ It returns the eperson that can claim the task. See the [eperson endpoint for mo
 
 It returns the group of epersons that can claim the task. See the [group endpoint for more info](epersongroups.md). This is a **read-only** endpoint, once the task is created the backend group cannot be changed.
 
+#### workflow step
+**/api/workflow/claimedtasks/<:id>/step** (READ-ONLY)
```

Comment:
@benbosman : Was just about to merge this when I noticed another tiny copy & paste error.  This URL should be `/api/workflow/pooltasks/<:id>/step`  (not claimedtasks)

Reply:
Good catch, I've updated it as well now